templates/all.html

```diff
@@ -0,0 +1,121 @@
+<form class="form" action="" id="form-{{sound_track}}">
+  <!-- mood tasks -->
+  <div class="box">
```

Comment:
I don't like duplicating this data in all.html and mood-miscellaneous.html. I would prefer to turn this into a macro (like the [checked macro](https://github.com/MTG/mtg-jamendo-annotator/blob/fc4818fb550758a2116ddcaee20480f581b513e6/templates/index.html#L1)) in order to be able to share the data between combined templates and individual templates. This will prevent error occurring from when we might change something in one place but not the other.